### PR TITLE
Create User Stories

### PR DESCRIPTION
**Pull request by jctrainor**
# Description
User stories document, which can also be found on teams Trello board

Fixes # (issue)

## Type of change

Documentation

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as
      expected)
- [X] Documentation (update or new)

# How Has This Been Tested?

## Testing Checklist:

- [X] Tested in latest Chrome
- [ ] Tested in latest Safari
- [X] Tested in latest Firefox

# Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code in hard-to-understand areas
- [X] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [X] I have requested a review from Liz Lynch on the Pull Request

### CODE REVIEW

File: docs/OnTrack/Deployment/User Stories [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+As a user, I want to be able to reset my Ontrack password myself, so I don&#39;t need to contact a system administrator.
+
+As a system administrator, I want users to be able to reset their passwords without my input. I also want the ability to send a password reset request to users. This will save me time and increase security by allowing insecure/exposed passwords to be easily changed.
+
+ **Thoth Tech Ontrack deployment to Google Cloud**
+
+As a product owner, I want to have more freedom in OnTrack development of the company&#39;s evolving vision separated from the Deakin version.
+
+As a student/admin/Doctors, I want to access the OnTrack via the URL from anywhere and should be able to perform all actions those are performed by students on Deakin version of track.
```
Comment (ginge270) [inline docs/OnTrack/Deployment/User Stories:31]: is this meant to have Doctors?

File: docs/OnTrack/Deployment/User Stories [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+As a product owner, I want separation of roles as available in Deakin version of OnTrack.
+
+As a product owner, I want an email sent every time there is a successful build sent to production
+
+As a product owner, I want cloud hosted version should implement the proxies.
```
Comment (ginge270) [inline docs/OnTrack/Deployment/User Stories:37]: i want *a* cloud hosted version *that* should implement the proxies?

**Comment by talzid** [comment]
@jctrainor just a slight nudge of this PR

**Comment by jctrainor** [comment]
PR has been updated

**Review by ginge270 — APPROVED** [review]

**Pull request merged**